app/renderer/lib/client-frameworks/java.js

```diff
@@ -97,6 +97,14 @@ ${this.indent(code, 4)}
   codeFor_back () {
     return `driver.navigate().back();`;
   }
+
+  codeFor_tap (varNameIgnore, varIndexIgnore, x, y) {
+    return `(new TouchAction(driver)).tap(${x}, ${y}).perform()`;
+  }
+
+  codeFor_swipe (varNameIgnore, varIndexIgnore, x1, y1, x2, y2) {
+    return `(new TouchAction(driver)).press({x: ${x1}, y: ${y1}}).moveTo({x: ${x2}: y: ${y2}}).release().perform()`;
```

Comment:
i think we should also split this onto multiple lines and indent for the generated code as well.